tests/storage/outofprocess_test.py

```diff
@@ -967,61 +986,56 @@ def test_glob(oop_cleanup, tmpdir):
 
 # External APIs
 
-def test_read_lines(oop_cleanup, tmpdir):
+def test_read_lines(oop_cleanup, user_mount):
     iop = oop.getProcessPool("test")
-    path = str(tmpdir.join("file"))
```

Comment:
This is the only change need in this test:

    path = os.path.join(user_mount, "file"))

The rest of the does not need any change, and the new version is actually worse, since we need to do str(user_ount) each time we use the patch, instead of once.
If ioprocess interface was accepting pathlib.Path, this could be better, but I don't think it worth our time to change the interface to support pathlib. This bring zero value to oVirt users and questionable value to developers.

Reply:
I agree that having to cast to string every Path is not so nice, but as you pointed out, my thought on this was that overtime `ioprocess` could accept Path. Value for developers could be to have OOP path handling and more readable code, and well, flexibility. But this I guess is a matter of taste mostly, and `os.path` will not be deprecated any time soon, if ever. So I will change the code back to it.